docs/site/tutorials/todo-list/todo-list-tutorial-relations.md

```diff
@@ -87,11 +140,57 @@ export class Todo extends Entity {
 
 #### Inclusion of Related Models
 
-When we ran the `lb4 relation` command, we accepted the default of `Yes` to the
-prompt:
+You'll notice there's a `TodoRelations` interface in the `Todo` model class file
+as well as a `TodoWithRelations` type defined.
+
+{% include code-caption.html content="src/models/todo.model.ts" %}
+
+```ts
+export interface TodoRelations {
+  // describe navigational properties here
+}
+
+export type TodoWithRelations = Todo & TodoRelations;
+```
+
+In the `TodoRelations` interface, we want to describe a `todoList` as a
+navigational property. We can do that as follows:
+
+{% include code-caption.html content="src/models/todo.model.ts" %}
+
+```ts
+// add TodoListWithRelations to the following import
+import {TodoList, TodoListWithRelations} from './todo-list.model';
+```
+
+```ts
+export interface TodoRelations {
+  // add the following line
+  todoList?: TodoListWithRelations;
+}
+```
+
+Let's add a `todo` navigational property to the `TodoList` model as well:
+
+{% include code-caption.html content="src/models/todo-list.model.ts" %}
+
+```ts
+// add TodoWithRelations to the following import
+import {Todo, TodoWithRelations} from './todo.model';
+```
+
+```ts
+export interface TodoRelations {
+  // add the following line
+  todos?: TodoWithRelations[];
+}
+```
+
+Further, when we ran the `lb4 relation` command, we accepted the default of
+`Yes` to the prompt:
 
 ```sh
-? Allow Order queries to include data from related Customer instances? (Y/n)
+? Allow Todo queries to include data from related TodoList instances? (Y/n)
```

Comment:
Do we want to add the hasOne relation page as the next page at [### Navigation](https://github.com/strongloop/loopback-next/pull/4412/files#diff-66a93d2a313a3c5eb51cc97687782df4R252)?

```
Previous step: [Add TodoList Repository](todo-list-tutorial-repository.md)
Last step: [Add TodoList Controller](todo-list-tutorial-controller.md)
```
->
```
Previous step: [Add TodoList Repository](todo-list-tutorial-repository.md)
Next step: [Add Has One Reltaion](todo-list-tutorial-has-one-relation.md)
```
And also add navigation links to `has-one.md`.